addons/pos_restaurant/static/src/js/models.js

```diff
@@ -151,59 +125,9 @@ patch(PosGlobalState.prototype, "pos_restaurant.PosGlobalState", {
         const ordersResponse = await this._save_to_server(ordersToSync, { draft: true });
         const tableOrders = [...this.ordersToUpdateSet].map((order) => order);
         ordersResponse.forEach((orderResponseData) =>
-            this._updateTableOrder(orderResponseData, tableOrders)
+            this._updateOrder(orderResponseData, tableOrders)
         );
     },
-    // created this hook for modularity
-    _updateTableOrder(ordersResponseData, tableOrders) {
```

Comment:
You can't simply remove this without checking if this is called elsewhere. There's even a comment explaining it has been made for modularity which means override. It is being used in `l10n_de_pos_cert` module and you're breaking the module

Reply:
I didn't check if the other methods you removed were being called elsewhere. 
Verify if you didn't remove anything that is still called elsewhere or the signature